advance/deleteDid.ts

```diff
@@ -0,0 +1,27 @@
+import { ConfigService, DidHelpers } from '@kiltprotocol/sdk-js'
+import { DidDocument, KiltAddress, SignerInterface } from '@kiltprotocol/types'
+
+export async function deleteDid(
+  didDocument: DidDocument,
+  signers: SignerInterface[],
+  submitter: SignerInterface<'Ed25519', KiltAddress>
+): Promise<void> {
+  const api = ConfigService.get('api')
+
+  const transactionHandler = DidHelpers.deactivateDid({
+    api,
+    signers: [...signers],
+    submitter,
+    didDocument: didDocument,
+  })
+
+  const result = await transactionHandler.submit()
```

Comment:
it's a little cleaner if call it like `const result = await DidHelpers.deactivateDid(...).submit()`, respecting the builder pattern. Breaking this up in two intermediate steps is a bit odd.

Reply:
Yes, I agree. Once I have all the branches merged. I will do another PR that will clean up all the supports.